code/controllers/configuration/entries/game_options.dm

```diff
@@ -181,6 +181,12 @@
 
 /datum/config_entry/string/alert_red_downto
 	config_entry_value = "The station's destruction has been averted. There is still however an immediate serious threat to the station. Security may have weapons unholstered at all times, random searches are allowed and advised."
+	
+/datum/config_entry/string/alert_gamma
```

Comment:
You need to add these to the config too :), not just here

Reply:
does it look like i can add things to the config

Reply:
you need to add it to the .txt here on the git at least

Reply:
🤔